docs/policy/software-support.md

```diff
@@ -1,143 +1,149 @@
-
-    To avoid any confusion around ticket ownership, assign only one Technology team member per ticket.
-    If the expertise of another Technology team member is needed on the ticket, add them to the CC list.
-    Inasmuch as possible, you should strive to handle the easier tickets and not pass them off to other people.
-
-### LCMAPS VOMS Transition ###
-
-This section contains the process for helping sites transition from edg-mkgridmap or GUMS to the LCMAPS VOMS plugin as
-part of the [VOMS Admin Server retirement](/policy/voms-admin-retire).
-
-1. Ask if the site is using edg-mkgridmap or GUMS. If they're using GUMS, find out what GUMS clients they have at their 
-   site. Possibilities include:
-
-    - HTCondor-CE
-    - GridFTP
-    - XRootD: if an ATLAS site, they use vomsxrd/xrootd-voms-plugin
-    - dCache: if an ATLAS site, encourage them to consult US-ATLAS mailing lists
-    - BeStMan: encourage the site to transition to GridFTP, as they cannot retire GUMS until BeStMan doesn't depend on it.
-      If CMS/ATLAS, encourage them to consult their US-ATLAS/US-CMS mailing lists for assistance.
-
-1. Ask them to follow the relevant instructions for their authorization solution 
-
-    - [edg-mkgridmap](http://www.opensciencegrid.org/docs/security/lcmaps-voms-authentication/#migrating-from-edg-mkgridmap)
-    - [GUMS](http://www.opensciencegrid.org/docs/security/lcmaps-voms-authentication/#migrating-from-gums)
-
-1. After they've completed the above instructions, use one of the following methods (in order of preference) to verify 
-   LCMAPS VOMS mappings:
-
-    - If the host is a CE, verify that they are still receiving pilots:
-
-        - Query their CE directly:
-
-                :::console
-                $ condor_q -name <CE HOSTNAME> -pool <CE HOSTNAME>:9619
-
-            This may not work if the site has a strict firewall or do not run an HTCondor-CE.
+-   We are a small community and we need to take good care of our users.
+    Please be friendly and patient even when the user is frustrated or lacking in knowledge.
+-   Always sign your ticket with your full name, so people know who is responding.
+    If it's easy for you, include a signature at the bottom of your response.
+-   If you need to collect information about a problematic host, ask users to run `osg-system-profiler`.
+    It can shorten the number of times you ask for information because it collects quite a bit for you.
+-   If you run across a problem that has a chance of being hit by other users, consider:
+    -   Is there a bug we should fix in the software?
+        Or something we could improve in the software?
+    -   Is there a way to improve our documentation?
+    -   Can you extend our troubleshooting documents to help people track this down more quickly?
+        Consider the troubleshooting documents to be as much for us as for our users.
+    -   Noting it during the support discussion during the weekly OSG Technology meeting
```

Comment:
The tense in this sentence doesn't match the other bullet points. Maybe unindent it and say "- Also consider noting it...", or just drop it altogether (since talking about support when you're asked about support is pretty obvious).